tests/test_wagon.py

```diff
@@ -753,16 +681,11 @@ def test_fail_create(self):
             _parse('wagon create non_existing_package -v -f')
         assert 'Failed to retrieve info for package' in str(ex)
 
-    @mock.patch('wagon.IS_VIRTUALENV_INSTALLED', False)
-    def test_create_with_validation_no_virtualenv_installed(self):
```

Comment:
This is again mocking out the only moving part

also btw, hopefully soon we'll be able to move to using stdlib venv instead of virtualenv, and then it'll be always installed.

Currently we're not using venv because it appears that on travis, venv is always making a `--system-site-packages --without-pip` venv. No idea why. Doesn't happen on any system that I have access to. And I don't have access to travis, because you have to ask their support to enable ssh.